source/_drafts/Introducing-Video-js-HTTP-Streaming-VHS.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Introducing Video.js HTTP Streaming (VHS)
+author:
+  name: Garrett Singer
+  github: gesinger
+tags:
+  - vhs
+---
+
+"How do I get my video to play with Video.js?"
+
+This is one of the most frequent questions we get when working on Video.js. And it's a good question.
+
+If someone checks out a copy of Video.js, their content may play on one browser and not another. To get cross browser support for a specific type of content, they must stumble upon a relevant source handler (for instance, [videojs-contrib-hls](https://github.com/videojs/videojs-contrib-hls) for HLS content). Only after including the source handler will their video play across all major browsers.
+
+We want Video.js to be easy to use, and that isn't the easiest setup. So we decided that it's time to address the issue.
+
+This was the motivation behind integrating [Video.js HTTP Streaming](https://github.com/videojs/http-streaming) (nicknamed VHS) inside of Video.js 7 by default, and we wanted to take a moment to describe what VHS is, and how it's different from including source handlers and plugins like before.
+
+# What is VHS?
```

Comment:
might want to make this H2 headers, based on how it looks in the Netlify preview

Reply:
yeah, the main title is `#`, sections should be `##`